### DISCUSSION TITLE
Google Analytics Import successful but not showing on chart

### DISCUSSION THREAD

**Discussion by DigNZ** [plausible/analytics#2659] [Self-Hosted Support]
I've spun up Plausible and its working great. I have 4 domains on it - 2 of which used to have Google Analytics. 

I connected up to Google and imported the data, for one domain it works perfectly. For the other, the summary numbers, sources, search terms, countries etc., are all perfect but the main chart only shows Plausible collected data.

I've toggled the 'G' on the chart to try and get it working but no luck. Also Unique Visitors is 0 for historical data, even though there are page views

It says 131k records imported.

![image](https://user-images.githubusercontent.com/817657/217698773-d0a40d77-7d1b-4f12-ba96-9270df6234f6.png)

See when I installed Plausible? :)
![image](https://user-images.githubusercontent.com/817657/217698831-4c2cce68-f336-4da1-8457-4be71dfee424.png)

**Comment by qodeboy** [comment]
Same issue here, did you manage to solve it?
